src/components/navigation-bar/navigation-bar.component.js

```diff
@@ -17,6 +18,7 @@ const NavigationBar = ({
       aria-label={ariaLabel}
       navigationType={navigationType}
       data-component="navigation-bar"
+      stickyOffset={stickyOffset}
       {...props}
```

Comment:
I think `stickyPosition` should be listed here instead of being passed as a content of `props`. That will be just more readable